openssl-dynamic/src/main/java/io/netty/tcnative/jni/SSL.java

```diff
@@ -656,4 +659,66 @@ public static native void setCertificateBio(
      * Free x509 chain ({@code STACK_OF(X509)} pointer).
      */
     public static native void freeX509Chain(long x509Chain);
+    
+    // Extracted from https://github.com/openssl/openssl/blob/master/include/openssl/tls1.h
+    
+    /**
+     * This is the default (initial) status type value for {@link SSLEngine}s.
+     */
+    public static final int TLSEXT_STATUSTYPE_host_name = 0;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^([a-z][a-zA-Z0-9]*|[A-Z][A-Z0-9_v]*)$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00115)